src/components/Routes/Routes.js

```diff
@@ -1,22 +1,25 @@
 import React from 'react'
-import { Route } from 'react-router-dom'
+import { Route, Switch, Redirect } from 'react-router-dom'
 import { Router } from 'react-router'
 import JobList from '../../pages/JobList'
-import JobEdit from '../../pages/JobEdit'
-import JobView from '../../pages/JobView'
 import JobAdd from '../../pages/JobAdd'
 import SequenceAdd from '../../pages/SequenceAdd'
 import SequenceEdit from '../../pages/SequenceEdit'
 import history from '../../services/history'
+import JobViewOrEdit from './JobViewOrEdit'
 
 const Routes = () => (
     <Router history={history}>
-        <Route exact path="/" component={JobList} />
-        <Route path="/edit/:id" component={JobEdit} />
-        <Route path="/view/:id" component={JobView} />
-        <Route path="/add" component={JobAdd} />
-        <Route path="/add-sequence" component={SequenceAdd} />
-        <Route path="/edit-sequence/:id" component={SequenceEdit} />
+        <Switch>
+            <Redirect from="/edit/:id" to="/job/:id" />
+            <Redirect from="/view/:id" to="/job/:id" />
+            <Redirect from="/add" to="/job/add" />
```

Comment:
What about `/add-sequence -> /queue/add` and `/edit-sequence/:id -> /queue/:id`?

Reply:
Those routes are not referenced anywhere yet, so noone should be using them. On a related note, the scheduler -> queue renaming thing I'm going to address in this issue: https://dhis2.atlassian.net/browse/DHIS2-15671